### DISCUSSION TITLE
[Question] Trying to model a Deformable linear object - the simulation is not very stable

### DISCUSSION THREAD

**Discussion by nadia-97** [isaac-sim/IsaacLab#1633] [Q&A]
### Question
![Screenshot from 2024-09-25 10-54-24](https://github.com/user-attachments/assets/398869bb-b049-42f1-97e8-9bde42be0778)
as you can see in the image, I created a cylindrical shaped long deformable object but upon interacting with the physics of the enviroments it acts strange, it seems to change the shape completely.

**Comment by mpgussert** [comment]
We would need more details about the deformable object and to quantify "change the shape completely" a bit more in order to assist you.  Sorry!  What do you mean by this? Does it clip into the table?  or is it too deformable and not holding it's shape? etc...

Thanks!

**Comment by nadia-97** [comment]
So I am trying to model a thin cylindrical object like a cable and the cyclinder doesn't hold shape, as you can see in the image it takes the shape of a conical figure even without any sort of manipulation applied to it.

**Comment by RandomOakForest** [comment]
Thank you for posting this. I'm moving this to a discussion for the team to follow up.
